samples/batch/transcription-enabled-storage/Connector/DatabaseConnector.cs

```diff
@@ -0,0 +1,254 @@
+﻿// <copyright file="DatabaseConnector.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Data.SqlClient;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Logging;
+
+    public class DatabaseConnector : IDisposable
```

Comment:
What is the databased used for? To store all results, or just meta data? I would think to store recognized result in Azure Storage and metadata in AzureTable. Do you really need database for that?

Reply:
I need the Database for PowerBI analytics - but this feature is optional